package/MDAnalysis/analysis/utils/autocorrelation.py

```diff
@@ -21,44 +21,120 @@
+the intermittent autocorrelation. The function :func:`autocorrelation`
+calculates the continuous autocorrelation only. The data may be
+pre-processed using the function :func:`intermittency` in order to
+acount for intermittency before passing the results to
+:func:`autocorrelation`.
+
+See Gowers and Carbonne, 2015, (DOI:10.1063.1.4922445) for a further
+discussion on the time continuous and intermittent autocorrelation of
+hydrogen bond lifetimes.
+
+**Analysis tools that make use of modules**
+
+* :class:`MDAnalysis.analysis.waterdynamics.SurvivalProbability`
+    Calculates the continuous or intermittent survival probability
+    of an atom group in a region of interest.
+
+* :class:`MDAnalysis.analysis.hbonds.hbond_analysis`
+    Calculates the continuous of intermittent hydrogen bond
+    lifetime.
+
+"""
+
 from __future__ import absolute_import, division
 import numpy as np
 from copy import deepcopy
 
 
 def autocorrelation(list_of_sets, tau_max, window_step=1):
-    r"""The descrete implementation of the autocorrelation function.
+    r"""Implementation of a discrete autocorrelation function.
+
+    The autocorrelation of a property :math:`x` from a time :math:`t=t_0` to :math:`t=t_0 + \tau`
+    is given by:
+
+    .. math::
+        C(\tau) = \langle \frac{ x(t_0)x(t_0 +\tau) }{ x(t_0)x(t_0) } \rangle
+
+    where :math:`x` may represent any property of a particle, such as velocity or
+    potential energy.
+
+    This function is an implementation of a special case of the time
+    autocorrelation function in which the property under consideration can
+    be encoded with indicator variables, :math:`0` and :math:`1`, to represent the binary
+    state of said property. This special case is often referred to as the
+    survival probability (:math:`S(\tau)`). As an example, in calculating the survival
+    probability of water molecules within :math:`5 \unicode{x212B}` of a protein, each water
+    molecule will either be within this cutoff range (:math:`1`) or not (:math:`0`). The
+    total number of water molecules within the cutoff at time :math:`t_0` will be
+    given by :math:`N(t_0)`. Other cases include the Hydrogen Bond Lifetime as
+    well as the translocation rate of cholesterol across a bilayer.
+
+    The survival probability of a property of a set of particles is
+    given by:
 
     .. math::
-       C_{HB}^c(\tau) = \frac{\sum_{ij}h_{ij}(t_0)h'_{ij}(t_0+\tau)}{\sum_{ij}h_{ij}(t_0)}
+        S(\tau) =  \langle \frac{ N(t_0, t_0 + \tau )} { N(t_0) }\rangle
+
+    where :math:`N(t0)` is the number of particles at time :math:`t_0` for which the feature
+    is observed, and :math:`N(t0, t_0 + \tau)` is the number of particles for which
+    this feature is present at every frame from :math:`t_0` to :math:`N(t0, t_0 + \tau)`.
+    The angular brackets represent an average over all time origins, :math:`t_0`.
+
+    See Araya-Secchi et al., 2014, (https://doi.org/10.1016/j.bpj.2014.05.037)
```

Comment:
Cite with reST.

Reply:
If use of this function should cite a paper then you can use a `duecredit` decorator. I don't know the history of the function, so your call.